static/js/util/profile_edit_test.js

```diff
@@ -268,21 +272,21 @@ describe('Profile Editing utility functions', () => {
       that.props.profile.date_of_birth = "1917-11-07";
       rerender(false);
       yearTextField.props.onChange({target: {value: "1233"}});
```

Comment:
can you set a `futureYear` and `tooFarInPastYear` (et al) variable near `currentYear` and `cutoff` and replace the hardcoded values (eg: "1233", "2077") with those?